index.js

```diff
@@ -101,11 +104,29 @@ class I18n {
                 }
                 break;
             default:
-                this[MISSING].forEach((fn) => fn(key, this.$scope, this.translations));
-                return I18n.getDefault(key);
+                this[MISSING].forEach((fn) => fn(`${key}`, this.$scope, this.translations));
```

Comment:
Is the template literal thing for stringification?

Reply:
Yes. It is used instead of `.toString()` as a safer way.
`key` can be an array - in which case they both turn to `'one,two'`
But if `key` happens to be unassigned it will print `'undefined'` where `.toString()` will throw an error.